.rubocop_todo.yml

```diff
@@ -209,21 +209,6 @@ Minitest/RefutePredicate:
 Minitest/TestMethodName:
   Enabled: false
 
-# Offense count: 2
-# This cop supports safe autocorrection (--autocorrect).
-Rake/Desc:
-  Exclude:
-    - 'Rakefile'
-    - 'lib/tasks/config.rake'
-
-# Offense count: 15
-Rake/MethodDefinitionInTask:
-  Exclude:
-    - 'Rakefile'
-    - 'infinite_tracing/tasks/proto.rake'
-    - 'lib/tasks/config.rake'
-    - 'lib/tasks/tests.rake'
-
```

Comment:
Seeing big removals like this are certainly a nice aspect of having the todo file. Nice!

Reply:
It is refreshing isn't it. I'm glad we when the todo file route!